### PR TITLE
AAP-1972: Add subsection on active offline token

### PR DESCRIPTION
**Pull request by lmalivert**
Created a subsection in AH Getting Started Guide - _1.2 Keeping your offline token active_

Jira issue: [AAP-1972](https://issues.redhat.com/browse/AAP-1972)

Preview link: [Keeping your offline token active](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/352)

### CODE REVIEW

File: downstream/modules/automation-hub/con-offline-token-active.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+[id="con-offline-token-active"]
```
Comment (oraNod) [inline downstream/modules/automation-hub/con-offline-token-active.adoc:2]: Can we remove "con-", "ref-", and "proc-" fragments from IDs? They aren't meaningful and end up as part of the URL which isn't great for SEO.

File: downstream/modules/automation-hub/con-offline-token-active.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+= Keeping your offline token active
+
+Your offline token will never expire by default and is not the subject of SSO Session Idle timeout and SSO Session Max lifespan. The offline token is valid even after a user logout or server restart. However, by default the token will expire after 30 days of inactivity.
```
Comment (oraNod) [inline downstream/modules/automation-hub/con-offline-token-active.adoc:6]: Last sentence contradicts "token will never expire by default". Also should we spell out SSO? Is this referring to Red Hat single sign-on / Keycloak?

Funnily enough idle time out and max lifespan are Infinispan / Data Grid parameters that configure how long tokens are stored in the system (RHSSO relies on RHDG as a data store). I'm not sure we really need to explain that level of detail here. It's probably too lowlevel to mention the underlying implementation details. How about possibly condensing this down a little?

Offline tokens expire after 30 days of inactivity. You can periodically refresh offline tokens to prevent them from expiring, which means users need to request a new offline token. (Not sure about the 2nd part about requesting a new token but it might be worth explaining the benefit of preventing tokens from expiring.)

HTH.

Comment (lmalivert) [reply downstream/modules/automation-hub/con-offline-token-active.adoc:6]: Hi @oraNod. Thanks for the comments/suggestions. This PR is a WIP so not quite ready for a peer-review just yet (sorry about the notification). I was troubleshooting an issue and had to create a PR to do that. 

Your comments and suggestions are valid. I'll revise/edit accordingly after the SME review :)

Comment (oraNod) [reply downstream/modules/automation-hub/con-offline-token-active.adoc:6]: Sorry for jumping the gun with the peer review @lmalivert Hope you have a great weekend.

File: downstream/modules/automation-hub/con-offline-token-active.adoc (hunk 1 above)
Comment (lmalivert) [reply downstream/modules/automation-hub/con-offline-token-active.adoc:2]: Yes, I'll remove it. Thanks @oraNod!

File: downstream/modules/automation-hub/con-offline-token-active.adoc (hunk 2 above)
Comment (lmalivert) [reply downstream/modules/automation-hub/con-offline-token-active.adoc:6]: No problem :) Honest mistake. Thanks for adding the new label 👍🏽

File: downstream/modules/automation-hub/con-offline-token-active.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+[id="con-offline-token-active"]
+
+= Keeping your offline token active
+
+Keeping an offline token active is useful in the event that an application needs to perform action on behalf of the user, even when the user is offline. For example, a routine data backup.
```
Comment (dcdacosta) [inline downstream/modules/automation-hub/con-offline-token-active.adoc:6]: Suggest rewording to simplify:
Keeping an offline token active is useful when an application needs to perform....

File: downstream/titles/hub/getting-started/master.adoc [hunk 4]
```diff
@@ -20,6 +20,8 @@ include::automation-hub/proc-create-api-token.adoc[leveloffset=+1]
 
 The API token is now available to use to configure {HubName} as your default collections server or when uploading collections using the `ansible-galaxy` command line tool.
```
Comment (dcdacosta) [inline downstream/titles/hub/getting-started/master.adoc:21]: This sentence is a bit awkward, suggest rewording to:

The API token is now available for configuring {HubName} as your default collections server or uploading collections using the `ansible-galaxy` command line tool.

**Review by dcdacosta — APPROVED** [review]
Two rephrasing suggestions for simplification, otherwise, LGTM.

File: downstream/titles/hub/getting-started/master.adoc (hunk 4 above)
Comment (lmalivert) [reply downstream/titles/hub/getting-started/master.adoc]: Thanks for the peer review @dcdacosta. I've made the changes. Can you pls approve and merge?

**Comment by oraNod** [comment]
@lmalivert Hi, hope you don't mind me commenting on this but I spotted `[WIP]` in the title. Perhaps it would be better to remove that before merging?

**Comment by lmalivert** [comment]
@oraNod Thanks for catching that. I removed the WIP.
